File: Algorithm/Programmers/Python/Level 1/2016년.py
def solution(a, b):
    lst = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]
    day = 0
    if a != 1:
        for i in range(1, a):
            day += days(i)
    day += b - 1
    return lst[day % 7]


def days(m):
    if m == 2:
        return 29
    elif m >= 1 and m <= 7:
        if m % 2 == 1:
            return 31
        else:
            return 30
    else:
        if m % 2 == 1:
            return 30
        else:
            return 31

File: Algorithm/Programmers/Java/Level 1/_2016년.java
public class _2016년 {
    public String solution(int a, int b) {
        String[] days={"FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"};
        int day=0;
        
        if(a==1){
            return days[(b-1)%7];
        }
        
        for(int i=1; i<a; i++){
            if(i==2){
                day+=29;
            }else if(i<8){
                if(i%2==0){
                    day+=30;
                }else{
                    day+=31;
                }
            }else{
                if(i%2==0){
                    day+=31;
                }else{
                    day+=30;
                }
            }
        }
        
        day+=b-1;
        return days[day%7];
    }
}
